dynamic-layers/meta-python/recipes-devtools/bcc/bcc_0.28.0.bb

```diff
@@ -22,7 +22,7 @@ LUAJIT:powerpc64 = ""
 LUAJIT:riscv64 = ""
 
 RDEPENDS:${PN} += "bash python3 python3-core python3-setuptools xz"
-RDEPENDS:${PN}-ptest = "cmake python3 python3-distutils python3-netaddr python3-pyroute2"
+RDEPENDS:${PN}-ptest = "cmake python3 python3-netaddr python3-pyroute2"
```

Comment:
Thanks for the patch I have a local change where I bump to 0.29.1 as well but ptests have failures so perhaps it will be good to first try it with existing rev and I have clang 18 as well which might be playing part too